lua/autorun/client/bounties.lua

```diff
@@ -0,0 +1,131 @@
+
+function NewBounty(b)
+
+	local bounty = vgui.Create("DFrame")
+	bounty:SetTitle("New Bounty")
+	bounty:SetSize(300, 144)
+	bounty:SetDeleteOnClose(true)
+	bounty:SetBackgroundBlur(true)
+	bounty:SetDrawOnTop(true)
+	
+	local l1 = vgui.Create("DLabel", bounty)
+	l1:SetText("Player:")
+	l1:Dock(TOP)
+	l1:DockMargin(4, 0, 4, 4)
+	l1:SizeToContents()
+
+	local pselect = vgui.Create("DComboBox", bounty)
+	pselect:SetValue("Select A Player")
+	pselect:SetTall(24)
+	pselect:Dock(TOP)
+
+	for _, ply in pairs(player.GetHumans()) do
+		pselect:AddChoice(ply:Nick(),ply:UniqueID())
```

Comment:
How is the UniqueID used? You can probably just net.WriteEntity the target player

Reply:
it may just be a leftover from previous versions of the code